### PR TITLE
Graduate JobMutableNodeSchedulingDirectives to GA

### PR DESCRIPTION
**Pull request by ahg-g**
Ref https://github.com/kubernetes/enhancements/issues/2926

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/job.md [hunk 1]
```diff
@@ -653,7 +646,7 @@ pod-to-node assignment to kube-scheduler. This is allowed only for suspended Job
 been unsuspended before.
 
 The fields in a Job's pod template that can be updated are node affinity, node selector, 
-tolerations, labels and annotations.
+tolerations, labels, annotations and scheduling gates.
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/job.md:649]: nit: can we make scheduling gates into a hyperlink?

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -281,6 +280,8 @@ For a reference to old feature gates that are removed, please refer to
 | `IdentifyPodOS` | `false` | Alpha | 1.23 | 1.23 |
 | `IdentifyPodOS` | `true` | Beta | 1.24 | 1.24 |
 | `IdentifyPodOS` | `true` | GA | 1.25 | - |
+| `JobMutableNodeSchedulingDirectives` | `true` | Beta | 1.23 | |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:283]:
```suggestion
| `JobMutableNodeSchedulingDirectives` | `true` | Beta | 1.23 | 1.26 |
```

**Review by sftim — COMMENTED** [review]
thanks!

File: content/en/docs/concepts/workloads/controllers/job.md (hunk 1 above)
Comment (ahg-g) [reply content/en/docs/concepts/workloads/controllers/job.md:649]: done.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 2 above)
Comment (ahg-g) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:283]: done.

File: content/en/docs/concepts/workloads/controllers/job.md [hunk 3]
```diff
@@ -653,7 +646,7 @@ pod-to-node assignment to kube-scheduler. This is allowed only for suspended Job
 been unsuspended before.
 
 The fields in a Job's pod template that can be updated are node affinity, node selector, 
-tolerations, labels and annotations.
+tolerations, labels, annotations and [scheduling gates](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-scheduling-readiness/).
```
Comment (tengqm) [inline content/en/docs/concepts/workloads/controllers/job.md:649]:
```suggestion
tolerations, labels, annotations and [scheduling gates]/docs/concepts/scheduling-eviction/pod-scheduling-readiness/).
```

Comment (ahg-g) [reply content/en/docs/concepts/workloads/controllers/job.md]: done, thanks

**Comment by tengqm** [comment]
Something is wrong, @ahg-g. This PR is not supposed to change 310 pages.

**Comment by ahg-g** [comment]
> Something is wrong, @ahg-g. This PR is not supposed to change 310 pages.

I fixed the rebase, thanks.

**Pull request merged**